pandas/io/json/table_schema.py

```diff
@@ -306,10 +306,9 @@ def parse_table_schema(json, precise_float):
         raise NotImplementedError('table="orient" can not yet read timezone '
                                   'data')
 
-    # No ISO constructor for Timedelta as of yet, so need to raise
-    if 'timedelta64' in dtypes.values():
-        raise NotImplementedError('table="orient" can not yet read '
-                                  'ISO-formatted Timedelta data')
+    for col, dtype in dtypes.items():
+        if dtype == 'timedelta64[ns]':
+            df[col] = df[col].apply(Timedelta)
```

Comment:
you would not convert this way, use ``pd.to_timedelta``

Reply:
Can `pd.to_timedelta` parse iso-formated timedeltas now? Should support for this format be added to this function?

Reply:
Should be able to change this now